yew/src/services/dialog.rs

```diff
@@ -41,4 +44,45 @@ impl DialogService {
             feature = "web_sys" => utils::window().confirm_with_message(message).unwrap(),
         }
     }
+
+    /// Prompts the user to input a message. In most browsers this will open an alert box with
```

Comment:
Could you add a note about what this function returns. For example, when does it return `None`? I think we might be able to add a test for that case btw

Reply:
Ooh, that's a good idea. I want to look into developing a set of integration tests as well so that we can test things which can only be tested in a browser.